xml/DialogPVRInfo.xml

```diff
@@ -201,6 +201,12 @@
 					<label>19165</label>
 					<visible>Window.IsActive(PVRGuideInfo)</visible>
 				</control>
+				<!-- Play programme -->
```

Comment:
Please rename to "Play recorded programme" for it to be clearer. Otherwise it sounds like it's redundant to the "switch" button.